sites/page-details/index.json-ld

```diff
@@ -32,55 +34,504 @@
+						"@value": "samples/page-details-3-en.html#share"
+					}
+				},
+				{
+					"@type": "source-code",
+					"@language": "fr",
+					"description": "Détails de la page avec Partagez cette page",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "samples/page-details-3-fr.html#share"
+					}
+				},
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Page details with Page feedback tool and Share this page",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "samples/page-details-3-en.html#pftShare"
+					}
+				},
+				{
+					"@type": "source-code",
+					"@language": "fr",
+					"description": "Détails de la page avec Outil de rétroaction sur la page et Partagez cette page",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "samples/page-details-3-fr.html#pftShare"
+					}
+				}
+			]
+		},
+		{
+			"@id": "_:iteration_pd_2",
+			"name": "Page details - Iteration 2",
+			"date": "2022-05",
+			"detectableBy": "footer.pagedetails",
+			"predecessor": "_:iteration_pd_1",
+			"successor": "_:iteration_pd_3",
+			"breaking": [
+				"Wrapping <div> changed to <footer>.",
+				"Changed grid sizes for RAP and Share this page.",
+				"RAP is now loaded through Data-AJAX with a fallback for basic mode."
+			],
+			"additions": [
+				"New invisible heading for accessibility."
+			],
+			"assets": [
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Code sample",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "samples/page-details-2-en.html"
+					}
+				},
+				{
+					"@type": "source-code",
+					"@language": "fr",
+					"description": "Exemple de code",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "samples/page-details-2-fr.html"
+					}
+				}
+			]
+		},
+		{
+			"@id": "_:iteration_pd_1",
+			"name": "Page details - Iteration 1",
+			"date": "2017-10",
+			"detectableBy": "div.pagedetails",
+			"successor": "_:iteration_pd_2",
+			"assets": [
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Code sample",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "deprecated/page-details-1-en.html"
+					}
+				},
+				{
+					"@type": "source-code",
+					"@language": "fr",
+					"description": "Exemple de code",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "deprecated/page-details-1-fr.html"
+					}
+				}
+			]
+		}
+	],
+	"changesets": [
+		{
+			"@id": "_:cs_pd_3",
+			"name": "Page details - Compatible with PFT + Accessiblity fix",
+			"status": "stable",
+			"baseOnIteration": "_:iteration_pd_3",
+			"detectableBy": "section.pagedetails",
+			"layout": [
+				"Page feedback tool (if included) on the top left",
+				"Share this page button (if included) on the top right",
+				"Date modified on bottom left"
+			],
+			"style": "Not applicable",
+			"semantic": {
```

Comment:
As discussed,

Move the include in a new property like "GCWeb template | includes"
And redefine this value for something basic like "section, h2" + a mapping of the static data + dynamic data (schema) if applicable.